### PR TITLE
moved lite_forms/generators to core/views

### PR DESCRIPTION
**Pull request by Tllew**
### Aim

[LTD-](https://uktrade.atlassian.net/browse/LTD-)

### CODE REVIEW

**Review by seijihg — APPROVED** [review]

**Review by saruniitr — APPROVED** [review]

File: core/views.py
Comment (kevincarrogan) [inline core/views.py:1]: I feel like just moving this file is sort of missing the point of why we want to get rid of `lite_forms`.

For example, this just moves `confirm_form` wholesale when in fact this should just be replaced or removed as it's kind of bad.

The same for `error_page` it's not really consistent with how we write views and should probably be changed as a result.

Comment (Tllew) [reply core/views.py:1]: So I thought about this for a while, and came to the conclusion that in the case of error_page it's a re_render and didn't really fit the new view/warrant a separate url, we have a set of handleSTATUSCODE urls, but the case for error_page in particular is to show an error page with a set message

I just wanted to move it to a more generic location as a first step.

Comment (kevincarrogan) [reply core/views.py:1]: I guess it's the fact though that a bunch of other `lite_forms` code got dragged across with `error_page` as well which made me feel like we weren't really solving any problems, we're just moving stuff so it doesn't say `lite_forms` in the directory name.

Comment (Tllew) [reply core/views.py:1]: this can wait to be fair, the rest is more important and removing the unused lite_form is pretty useful

**Pull request closed without merging**